File: aula8/exercicio_2/cpp_logic/exercicio2_procedural.cpp
#include <iostream>
using namespace std;


int main() {
    double base, altura, area;

    cout << "Digite a base do retangulo: ";
    cin >> base;
    cout << "Digite a altura do retangulo: ";
    cin >> altura;
    area = base * altura;
    cout << "Area do retangulo: " << area << endl;
    return 0;
}

File: aula8/exercicio_2/java_logic/Exercicio2Procedural.java
import java.util.Scanner;

public class Exercicio2Procedural {

    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        System.out.print("Digite a base do retangulo: ");
        double base = entrada.nextDouble();
        System.out.print("Digite a altura do retangulo: ");
        double altura = entrada.nextDouble();
        double area = base * altura;
        System.out.println("Area do retangulo: " + area);


        entrada.close();
    }
}